src/chapters/configuring.xml

```diff
@@ -3142,6 +3142,24 @@ ed272546-87bd-4db9-acba-e36e1a9ca20a
                                         sends ASCII code 127 (Delete) if this option is not set.</para>
                                 </entry>
                             </row>
+                            <row>
+                                <entry><parameter>terminal-type</parameter></entry>
+                                <entry>
+                                    <para><indexterm>
+                                            <primary>SSH</primary>
+                                            <secondary>terminal-type</secondary>
+                                        </indexterm>This parameter sets the terminal emulator type
+                                        string that is passed to the SSH server. Possible values
```

Comment:
I'm not sure it's worth listing out all of the possible values, here.  Since this is being documented for the Guacamole Client interface the values are in a drop-down box and the user will be able to see them.  It should be sufficient just to document the purpose and what happens if you don't specify it, and leave the list off.

Reply:
Fixed